DEVELOPER_ONBOARDING.md

```
@@ -0,0 +1,362 @@
+```elixir
+:killmails  # Hot killmail data (5 min TTL)
+:systems    # System data (1 hour TTL)
+:esi_data   # External API data (24 hour TTL)
+:temp_data  # Processing artifacts (5 min TTL)
+```
+
+#### 3. Unified Infrastructure
+
+- **HTTP**: Layered architecture with `Core.Http.Client` foundation
+- **Health**: Single `Health` module for all health checks
+- **Testing**: Unified `UnifiedTestCase` for all test types
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)